### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by vansosnin**
Hi!
Fixed several bugs with colorCase and trimLeadingZero.

### CODE REVIEW

File: src/__tests__/fixtures/color-case.compressed.css [hunk 1]
```diff
@@ -1 +1 @@
-p{color:#AA93FE;background-color:#AFE}
\ No newline at end of file
+p{color:#AA93FE;background:#FFF url(/ImagesDir/loader.gif) center center no-repeat}
```
Comment (ben-eb) [inline src/__tests__/fixtures/color-case.compressed.css]: This should be #fff if I'm not mistaken?

Comment (vansosnin) [inline src/__tests__/fixtures/color-case.compressed.css]: During exploring the code, I noticed that we do not want to apply transforms to `compressed` format... or we do want it? :)

Anyways, it does not transform right now. Same for other transforms that I've committed.

Comment (vansosnin) [inline src/__tests__/fixtures/color-case.compressed.css]: @ben-eb ping :)

Comment (ben-eb) [inline src/__tests__/fixtures/color-case.compressed.css]: If we are applying this transform to the other formats then it makes sense to do it here also, right?

Comment (vansosnin) [inline src/__tests__/fixtures/color-case.compressed.css]: Done!

**Comment by vansosnin** [comment]
Applied transformations to _compressed_ format.

File: src/__tests__/fixtures/color-case.compressed.css (hunk 1 above)
Comment (Mottie) [inline src/__tests__/fixtures/color-case.compressed.css]: This is also related to the issue #43 where a hex color + data URI is all transformed.

Comment (vansosnin) [inline src/__tests__/fixtures/color-case.compressed.css]: As I can see, the case from https://github.com/ben-eb/perfectionist/issues/43 is going to be fixed with this PR merge.

**Comment by ben-eb** [comment]
Thanks! 👍

**Pull request merged**